src/daemon/buildinfo.c

```diff
@@ -112,6 +112,7 @@ typedef enum __attribute__((packed)) {
     BIB_EXPORT_SHELL,
     BIB_DEVEL_TRACE_ALLOCATIONS,
     BIB_DEVELOPER_MODE,
+    BIB_LIB_WOLFSSL,
```

Comment:
Instead of adding a new build info entry for WolfSSL, could we modify the existing ‘OpenSSL’ one to have a name like ‘TLS Library’ and list the implementation being used?

We’re never going to support using more than one at a time, so this will let us keep the build info more concise.

Reply:
Done!